packages/screens/src/SettingsTemplate/SettingsTemplate.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { Button, Cell, Icon, IconName, Spacer, Typography } from "@equinor/mad-components";
+import { View, ScrollView } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+
+export type CellType =
+    | {
+          name: "navigation";
+          param: { route: string; params?: Record<string, any> };
+      }
+    | {
+          name: "button";
+          onPress: () => void;
+          color: Parameters<typeof Cell.Button>[0]["color"];
+      }
+    | {
+          name: "switch";
+          isActive: boolean;
+          onChange: (isActive: boolean) => void;
+          switchSize?: "small" | "normal";
+          disabled?: boolean;
+          description?: string;
+          color?: string;
+      }
+    | { name: "custom"; param: any };
```

Comment:
Lets have a discussion on what this custom type should provide after you are done looking at the other stuff in this PR